start.py

```diff
@@ -95,17 +99,20 @@ def contest_election(thisNode, listener):
 
     # print('contestableHosts nodes are: {}'.format(contestableHosts))
 
+    #Sending a request for election when more than one host is connected
     if len(contestableHosts) >= 1:
         for host in contestableHosts:
             propagate_election_thread = threading.Thread(target=listener.send_contest_request, args=("election", (host[0], 4242)))
             propagate_election_thread.start()
     else:
-        #update master and broadcast
+        #update master and broadcast information when only one host is connected
```

Comment:
Correct! But not only.  This is also so say if no other node with higher last IP octet, then I will appoint myself as the master